### PR TITLE
Feature/ayokunle/add user login

### PR DESCRIPTION
**Pull request by mrayor**
- User Login with email and password

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -15,4 +15,7 @@ web-report/
 # Yarn Integrity file
 .yarn-integrity
 /yarn.lock
-package-lock.json
\ No newline at end of file
+package-lock.json
```
Comment (parkerthegeniuschild) [inline .gitignore]: I don't know why you like to add package lock to gitignore . . we'll i guess to save space for PR requests . . . but you don't solve a headache by cutting of the head, lol . . . 
I just stopped reviewing after this line.

Comment (mrayor) [reply .gitignore]: I am just following the conventions provided. I can push the package-lock as a separate PR.

**Comment by enyeInc** [comment]
@mrayor you have a minor merge conflict to fix

File: .gitignore (hunk 1 above)
Comment (parkerthegeniuschild) [reply .gitignore]: Great thanks . .

**Comment by mrayor** [comment]
> @mrayor you have a minor merge conflict to fix

fixing it now

**Pull request merged**